pkgs/by-name/ki/kiwitalk/package.nix

```
@@ -0,0 +1,133 @@
+    pname = "${pname}-pnpm-deps";
+    inherit src version;
+
+    nativeBuildInputs = [
+      jq
+      moreutils
+      nodePackages.pnpm
+      cacert
+    ];
+
+    installPhase = ''
+      export HOME=$(mktemp -d)
+      pnpm config set store-dir $out
+      # cat package.json | jq '.pnpm.supportedArchitectures += { "os": ["linux"], "cpu": ["arm64"] }' | sponge package.json
+      pnpm install --frozen-lockfile --ignore-script
+
+      rm -rf $out/v3/tmp
+      for f in $(find $out -name "*.json"); do
+        sed -i -E -e 's/"checkedAt":[0-9]+,//g' $f
+        jq --sort-keys . $f | sponge $f
+      done
+    '';
+
+    dontBuild = true;
+    dontFixup = true;
+    outputHashMode = "recursive";
+    outputHash = {
+      x86_64-linux = "sha256-LJPjWNpVfdUu8F5BMhAzpTo/h6ax7lxY2EESHj5P390=";
+      aarch64-linux = "sha256-N1K4pV5rbWmO/KonvYegzBoWa6TYQIqhQyxH/sWjOJQ=";
+      x86_64-darwin = "sha256-9biFAbFD7Bva7KPKztgCvcaoX8E6AlJBKkjlDQdP6Zw=";
+      aarch64-darwin = "sha256-to5Y0R9tm9b7jUQAK3eBylLhpu+w5oDd63FbBCBAvd8=";
+    }.${stdenv.system} or (throw "Unsupported platform");
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "tauri-plugin-log-0.0.0" = "sha256-8BrFf7vheMJIaZD0oXpi8V4hmUJFzHJmkcRtPL1/J48=";
+      "tauri-plugin-single-instance-0.0.0" = "sha256-8BrFf7vheMJIaZD0oXpi8V4hmUJFzHJmkcRtPL1/J48=";
+    };
+  };
+
+  nativeBuildInputs = [
+    rustPlatform.cargoSetupHook
+    cargo
+    rustc
+    cargo-tauri
+    nodePackages.pnpm
+    copyDesktopItems
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+    libayatana-appindicator
+    webkitgtk
+  ];
+
+  preBuild = ''
+    export HOME=$(mktemp -d)
+    pnpm config set store-dir ${pnpm-deps}
+    pnpm install --offline --frozen-lockfile --ignore-script
+    pnpm rebuild
+    cargo tauri build -b deb
+  '';
+
+  preInstall = ''
+    mv target/release/bundle/deb/*/data/usr/ $out
+  '';
+
+  postInstall = ''
+    install -DT backend/bin/app/icons/512x512.png $out/share/icons/hicolor/512x512/apps/kiwi-talk.png
+    install -DT backend/bin/app/icons/256x256.png $out/share/icons/hicolor/256x256/apps/kiwi-talk.png
+    install -DT backend/bin/app/icons/128x128.png $out/share/icons/hicolor/128x128/apps/kiwi-talk.png
+    install -DT backend/bin/app/icons/32x32.png $out/share/icons/hicolor/32x32/apps/kiwi-talk.png
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "KiwiTalk";
+      exec = "kiwi-talk";
+      icon = "kiwi-talk";
+      desktopName = "KiwiTalk";
+      comment = "An UNOFFICIAL cross-platform KakaoTalk client";
+      categories = [ "Network" "InstantMessaging" ];
+    })
+  ];
+
+  meta = with lib; {
+    description = "An UNOFFICIAL cross-platform KakaoTalk client written in TypeScript & Rust (SolidJS, tauri)";
+    homepage = "https://github.com/KiwiTalk/KiwiTalk";
+    maintainers = with maintainers; [ honnip ];
+    license = licenses.asl20;
+    platforms = platforms.linux;
```

Considering you provide darwin hashes, should this be relaxed to
```suggestion
    platforms = platforms.linux ++ platforms.darwin;
```

?